main/src/library/FilePath.flix

```diff
@@ -155,3 +155,25 @@ pub eff FilePath {
     pub def list(f: String): List[String]
 
 }
+
+mod FilePath {
+
+    import java.io.File
+
+    ///
+    /// Handles the `FilePath` effect of the given function `f`.
+    ///
+    /// In other words, re-interprets the `FilePath` effect using the `IO` effect.
+    ///
+    pub def handle(f: a -> b \ ef): a -> b \ (ef - FilePath) + IO = x ->
+        try {
+            f(x)
+        } with FilePath {
+            def exists(f, k) = {
```

Comment:
Actually this is not totally correct. You need a try-catch around the block starting on this line.

Then you need to return a Result[FileError, b].